pkgs/by-name/fl/flow-editor/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  zig,
+  callPackage,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "flow";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "neurocyte";
+    repo = "flow";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-dXWqxV66BwtjOvmreq4+u5+xBI+1v1PAep8RQBK3rlA=";
+  };
+  postPatch = ''
+    ln -s ${callPackage ./build.zig.zon.nix { }} $ZIG_GLOBAL_CACHE_DIR/p
+  '';
+
+  nativeBuildInputs = [
+    zig.hook
+  ];
+
+  zigBuildFlags = [
+    "-Doptimize=ReleaseSafe"
```

Comment:
I thought it's ReleaseSafe by default? (I might be wrong)

Reply:
Yeah, I had not checked the build flags, I removed it